assets/js/components/Dashboard/AlertCard.tsx

```diff
@@ -0,0 +1,124 @@
+import moment from "moment";
+import React from "react";
+import { Col, Container, Row } from "react-bootstrap";
+import { ChevronRight } from "react-bootstrap-icons";
+import { ActivePeriod, Alert } from "../../models/alert";
+
+interface AlertCardProps {
+  alert: Alert;
+}
+
+const AlertCard = (props: AlertCardProps): JSX.Element => {
+  const { alert } = props;
+
+  const formatEffect = (effect: string) => {
+    return effect
+      .toLowerCase()
+      .replace("_", " ")
+      .split(" ")
+      .map((str: string) => str[0].toUpperCase() + str.substring(1))
+      .join(" ");
+  };
+
+  const renderEffect = (effect: string, severity: string) => {
+    const formattedEffect = formatEffect(effect);
+    if (effect.toLowerCase() === "delay") {
+      return `${formattedEffect}—${severity}`;
+    }
+
+    return formattedEffect;
+  };
+
+  const renderActivePeriod = (activePeriod: ActivePeriod[]) => {
+    const startDate = moment(activePeriod[0].start);
+    let result;
+
+    const startDateHtml = (
+      <div className="alert-card__alert-details__active-period-start">
+        {startDate.format("l")} · {startDate.format("LT")}
+      </div>
+    );
+
+    if (activePeriod[0].end) {
+      const endDate = moment(activePeriod[0].end);
+      result = (
+        <>
+          {startDateHtml}
+          <div className="alert-card__alert-details__active-period-end">
+            <span className="alert-card__alert-details__active-period__to">
+              to{" "}
+            </span>
+            <span>
+              {endDate.format("l")} · {endDate.format("LT")}
+            </span>
+          </div>
+        </>
+      );
+    } else {
+      result = (
+        <>
+          {startDateHtml}
+          <div className="alert-card__alert-details__active-period-end">
+            Until further notice
+          </div>
+        </>
+      );
+    }
+
+    return result;
+  };
+
+  return (
+    <div className="alert-card" data-testid={alert.id}>
+      <Container fluid className="alert-card__alert-details">
+        <Row>
+          <Col lg="auto">
+            {alert.affected_list.map((icon: string) => (
+              <img
+                className="alert-card__alert-details__pill"
+                key={`${alert.id}-${icon}`}
+                src={`/images/pills/${icon.toLowerCase()}.svg`}
+                alt={icon}
+              />
+            ))}
```

Comment:
Multiple route pills will need to stack vertically

![Screen Shot 2022-10-20 at 2 29 11 PM](https://user-images.githubusercontent.com/69368883/197029371-4035d8a7-e09a-4c88-b64a-d6ce7adf47fd.png)